xExtension-ShareByEmail/views/shareByEmail/share.phtml

```diff
@@ -3,75 +3,75 @@
 	/** @var ShareByEmail\mailers\View $this */
 ?>
 <div class="post">
-    <h1><?= _t('shareByEmail.share.title') ?></h1>
+	<h1><?= _t('shareByEmail.share.title') ?></h1>
 
-    <p>
-        <?= _t('shareByEmail.share.intro', isset($this->entry) ? $this->entry->title() : '') ?>
-    </p>
+	<p>
+		<?= _t('shareByEmail.share.intro', isset($this->entry) ? $this->entry->title() : '') ?>
+	</p>
 
-    <form class="sbe-form-share" method="POST">
-        <input type="hidden" name="_csrf" value="<?= FreshRSS_Auth::csrfToken() ?>" />
+	<form class="sbe-form-share" method="POST">
+		<input type="hidden" name="_csrf" value="<?= FreshRSS_Auth::csrfToken() ?>" />
 
-        <div class="form-group">
-            <label class="group-name" for="to">
-                <?= _t('shareByEmail.share.form.to') ?>
-            </label>
+		<div class="form-group">
+			<label class="group-name" for="to">
+				<?= _t('shareByEmail.share.form.to') ?>
+			</label>
 
-            <div class="group-controls">
-                <input
-                    type="email"
-                    name="to"
-                    id="to"
-                    class="long"
-                    placeholder="camille@dupuis-morizeau.com"
```

Comment:
What was that? 😮

Reply:
ping @marienfressinaud

Reply:
Oh that :D The "Dupuis Morizeau" is the fake family that we often use at Framasoft to illustrate our articles or documentation. It changes from the "John Doe" and similar names :)

Reply:
Ok. It could have been worse... But it is not a good practice to invent potentially valid domain names. Please check https://fr.wikipedia.org/wiki/Example.com

Reply:
Yes, I use `@example.org` or `@example.com` now, but I didn't know that these domains were reserved when I wrote this code :sweat_smile: Good thing that you changed it!